### DISCUSSION TITLE
Is Code app really a free download?

### DISCUSSION THREAD

**Discussion by infinitepower18** [thebaselab/codeapp#536] [Q&A] [answered]
In the changelog of Code app it is mentioned the main app is now free. But it is listed as a paid app on the App Store.

![IMG_4397](https://user-images.githubusercontent.com/44692189/183293795-659e9d8a-6b6a-483d-9da3-8191a409e773.PNG)

![IMG_4396](https://user-images.githubusercontent.com/44692189/183293805-019b6457-9fbc-4c44-89d4-1e478c4e16f9.PNG)

**Comment by hg0428** [comment]
Its free to install on Testflight

**Comment by bummoblizard** [comment, marked as answer]
It was free for a period of time. Regardless, you can still install it for free on TestFlight: https://testflight.apple.com/join/EgZ8sE2P

**Comment by utariq-com** [comment]
i cant connect to my server. its because we need option for passive mode or active mode. please implement this feature and update.

regards
